microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -34,6 +35,9 @@ Both approaches for hosting the WebView2 control in your app are similar in func
 
 There are instances where you might want to focus on displaying web content as quickly and easily as possible in your app.  Windowed hosting allows for a solution that quickly displays web content without having to include features for inputs, outputs, and accessibility.
 
+#### Window to Visual hosting: For a similar experience to Windowed hosting with some added benefits and tradeoff
+
+Window-to-Visual hosting is meant to be as easy to adopt as windowed hosting. It only requires setting an environment variable to enable. It does not require use of the visual hosting APIs. It resolves certain DPI issues that can result when sharing a WebView2 user data folder across applications. It does not support pen input and handwriting. To enable it, users must set this environment variable: `COREWEBVIEW2_FORCED_HOSTING_MODE` to the value: `COREWEBVIEW2_HOSTING_MODE_WINDOW_TO_VISUAL`.
```

Comment:
during Writer/Editor pass commit, i'll reduce number of "it"

Reply:
result re: final sentence:
To enable Window to Visual hosting, your code sets the environment variable `COREWEBVIEW2_FORCED_HOSTING_MODE` to the value `COREWEBVIEW2_HOSTING_MODE_WINDOW_TO_VISUAL`.